### PR TITLE
[Enhancement] Only handle charge Webhook's

### PR DESCRIPTION
**Pull request by wallaceSF**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| **Issue**         | https://mundipagg.atlassian.net/browse/PAA-157
| **What?**         |Now the webhook's charges are responsible for changing the order status and creating invoices.
| **Why?**          | Was having a lot of synchronization problems between the order and charge webhook
| **How?**          | The order handle is now called together with the charge handle

### CODE REVIEW

**Comment by pedrolucasmr** [comment]
Tem que ver esses problemas que apareceram no CodeClimate.

File: src/Kernel/Aggregates/Order.php [hunk 1]
```diff
@@ -103,6 +104,54 @@ public function getCharges()
...
+            return;
+        }
+
+        $listChargeStatus = [];
+        foreach ($this->getCharges() as $charge) {
+            $listChargeStatus[$charge->getStatus()->getStatus()] =
+                $charge->getStatus()->getStatus();
+        }
+
+        $chargesStatusEquals = false;
```
Comment (GabrielDeveloper) [inline src/Kernel/Aggregates/Order.php]: Acho que pode mudar para $chargesStatusEquals = count($listChargeStatus) == 1;

File: src/Kernel/Aggregates/Order.php [hunk 2]
```diff
@@ -103,6 +104,54 @@ public function getCharges()
...
+        ) {
+            $this->setStatus(OrderStatus::paid());
+            return;
+        }
+
+        if (
+            in_array(ChargeStatus::paid()->getStatus(), $listChargeStatus) &&
+            in_array(ChargeStatus::overpaid()->getStatus(), $listChargeStatus)
+        ) {
+            $this->setStatus(OrderStatus::paid());
```
Comment (GabrielDeveloper) [inline src/Kernel/Aggregates/Order.php:133]: [Duvida] Quando temos um cenário de 2 charges pagas, ele faz o setStatus pra paid aqui em outro lugar?

**Review by nmaia-zz — DISMISSED** [review]

**Review by GabrielDeveloper — CHANGES_REQUESTED** [review]

File: src/Kernel/Aggregates/Order.php (hunk 1 above)
Comment (wallaceSF) [reply src/Kernel/Aggregates/Order.php]: Boa. Feito!

File: src/Kernel/Aggregates/Order.php (hunk 2 above)
Comment (wallaceSF) [reply src/Kernel/Aggregates/Order.php:133]: no último if

Comment (wallaceSF) [reply src/Kernel/Aggregates/Order.php:133]: if (
            $chargesStatusEquals &&
            !$this->getCharges()[0]->getStatus()->equals(ChargeStatus::underpaid())
        ) {
            $currentStatus = reset($listChargeStatus);
            $this->setStatus(OrderStatus::$currentStatus());
        }

**Review by GabrielDeveloper — DISMISSED** [review]

**Review by nmaia-zz — DISMISSED** [review]

**Review by nmaia-zz — DISMISSED** [review]

**Review by GabrielDeveloper — APPROVED** [review]

**Review by michelpl — APPROVED** [review]

**Pull request merged**